README.mkd

```diff
@@ -32,6 +32,7 @@ environments and modules.
   Puppet 2.7 may work, but is generally not recommended as Puppet 2.7 is EOL.
   - Git is required for creating environments and modules from Git
   - SVN is required for creating environments and modules from SVN
+  - [Deploy Keys](https://developer.github.com/guides/managing-deploy-keys/#deploy-keys) (Git) or User Rights (SVN) for the r10k user in the source control system.
```

Comment:
This is an implementation detail for Git sources and should be pushed to (https://github.com/puppetlabs/r10k/blob/master/doc/dynamic-environments/git-environments.mkd).